### PR TITLE
Yherman training

### LINKED ISSUE #4: yherman planning for holiday
where should I go? 
- beach
- mountain
- city

### PR DESCRIPTION
**Pull request by yherman**
This pull request adds a file for things to do for holiday.
This PR fixes #4 
@githubteacher provide comment

### CODE REVIEW

**Comment by jray1110** [comment]
Sounds good to me

File: yherman-training.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Holiday Plan
+
```
Comment (jray1110) [inline yherman-training.md:2]: need to add another event here

File: yherman-training.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+### beach
```
Comment (saikatta) [inline yherman-training.md:3]: cool , but all beaches here are close

**Review by saikatta — COMMENTED** [review]
yours look good

File: yherman-training.md (hunk 2 above)
Comment (yherman) [reply yherman-training.md:3]: fly to south lol

**Review by jray1110 — CHANGES_REQUESTED** [review]
Need to add JD's BBQ

**Pull request merged**